healthcheck.go

```diff
@@ -53,6 +54,55 @@ type ClusterHealth struct {
 	Status string `json:"status"`
 }
 
+//indexcheck calls elasticsearch to check if the required indexes from the client exist
+func (cli *Client) indexcheck(ctx context.Context) (code int, err error) {
+
+	for _, index := range cli.indexes {
+
+		urlIndex := cli.url + "/" + index
+		logData := log.Data{"url": urlIndex, "index": index}
+
+		URL, err := url.Parse(urlIndex)
+		if err != nil {
+			log.Event(ctx, "failed to create url for elasticsearch indexcheck", log.ERROR, logData, log.Error(err))
+			return 500, err
+		}
+
+		path := URL.String()
```

Comment:
this variable is the same as `urlIndex`, I imagine that `url.Parse(urlIndex)` intention was just to validate that the urlIndex was valid, I think we should use `urlIndex` instead of `path`

Reply:
Removed this variable